daemon/k8s_watcher.go

```diff
@@ -372,11 +372,11 @@ func (d *Daemon) deleteK8sNetworkPolicyDeprecated(obj interface{}) {
 	})
```

Comment:
Let's add a field to include version of the object to all scopedLog so we don't need to differentiate between v1beta and GA